documentation/tutorials/buildlinux.rst

```diff
@@ -0,0 +1,58 @@
+=================================
+  How to build the Linux kernel
```

Comment:
Maybe something like this:

> How to build Linux Kernel for an x86_64 architecture.

The steps you described here are related to x86_64, and since we support cross-compile, I think it is a good idea to clarify that we are talking about x86.